### DISCUSSION TITLE
numpy vs python speed

### DISCUSSION THREAD

**Discussion by ripun-jay** [codewars/codewars.com#2853] [General]
![numpy vs panda speed](https://user-images.githubusercontent.com/84166592/200178047-9acc9560-e442-465a-80c7-1b611068a3cf.png)

WHY NUMPY IS SLOWER THEN PYTHON HERE
PLEASE LET ME KNOW

**Comment by hobovsky** [comment]
This is a very poor test which does not verify performance difference between numpy and "pure" pyton AT ALL.

**Comment by textninja** [comment]
You’re converting between data structures and depriving numpy the opportunity to make optimizations. Try this instead:

```python
import numpy as np
a = np.arange(1000000)
%timeit b = a*3
```
